app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :user, foreign_key: :user_id
```

Comment:
- [ ] Kindly check comments above and modified this

Reply:
I am not sure about this. But I just wanted to follow the ERD. and since my tables are created using that, I think I will have to change everything if I want to use author_id

Reply:
@FrederickMih